src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.tsx

```diff
@@ -66,6 +67,26 @@ const applyDataElementCodes = (code, dataElement) => {
   return updatedDataElement;
 };
 
+const applyNegationRationale = (code, dataElement) => {
+  const modelClass = getDataElementClass(dataElement);
+  const updatedDataElement = new modelClass(dataElement);
+  const cqlCode = new CQL.Code(
+    code.code,
+    code.system,
+    code.version,
+    code.display
+  );
```

Comment:
I think code is already an instance of CQL.Code. So, probably this can be avoided as well

Reply:
@adongare removed.